src/pages/preparing-for-tech-talk-part-3-content/index.md

```diff
@@ -27,7 +27,7 @@ There are two ways to build something.
 
 You can build **top-down**, where you start with a crude overall outline and then gradually refine each individual part. Or you can build **bottom-up**, starting with a small but polished fragment, and then growing everything else around it. This might remind you of how some image formats always load from top to bottom, while others start blurry at first but then get sharper as more data is loaded.
 
-I usually **combine these approaches** when preparing the talks.
```

Comment:
I think you're talking about talks in general here, not some talks specifically so we don't need `the`. Similar reasoning applies below.